### ISSUE TITLE
Nagios standard console not updated

### ISSUE THREAD

**Issue by quirinobrizi** [jbreams/nagmq#30]
Hi,

I compiled NagMQ module for Nagios 3 and Nagios 4 and integrated it, it seems to work correctly but when it is integrated, on both Nagios versions, the standard console is not updated although I see from the debug logs and status.dat that commands are issued and monitoring performed. 

Is that an expected behaviour?

Thanks,
~Q

**Comment by jbreams** [comment]
What do you mean by the standard console? What are you trying to do with nagmq?

**Comment by quirinobrizi** [comment]
Hi,

Thanks for you answer.

Whit standard console I mean the Nagios console that you access via HTTP. The fact that the HTTP console is updated is an important requirement in my case.

What I'm trying to do with NagMQ is expose the internal Nagios stati and events to a broker and from here forward hosts and services information to elasticsearch.

In both cases Nagios 3 and 4 I have a fresh install no custom configuration apart the one for "broker_module" which points to the compiled NagMQ module. The value configured for "event_broker_options" is -1, if I control what is sent to the broker via BROKER_* definition from include/broker.h, I can see the web console been updated depending on the value I use.

Clearly NagMQ has been compiled against Nagios 3 and 4 headers using the configure flags as described on the README file.

Thanks,
~Q

**Comment by jbreams** [comment]
I'm not sure what updates to the web interface you're expecting. The web interface doesn't interact with nagmq at all, it just uses the information in status.dat. Unless you're sending commands to the command (PULL) interface of nagmq, it won't change any data in Nagios. It sounds like you want to get information out of the publisher interface and have nagmq be passive - as just a way to get information out of nagios and into elastic search - what updates to the web interface are you expecting?

**Comment by quirinobrizi** [comment]
Hi,

The updates I'm expecting on the web interface are the basic: if a service transitions from i.e. OK to CRITICAL and/or vice versa, that should be reflected on the web interface, and it happens if I remove NagMQ as a listener of the Nagios broker, as soon as I add NagMQ as a listener from the broker the web interface does not show any update, to verify that I have installed SSHD in my local instance and started/stopped it. 

If I start/stop SSHD and NagMQ is not connected I see from the Nagios web interface that the status of SSH service changes from OK -> CRITICAL or vice versa.

If I start/stop SSHD and NagMQ is connected I see from the Nagios web interface that the status of SSH service does not changes from OK -> CRITICAL or vice versa.

In any case this is now more out of curiosity as I moved on using Nagios event handler to achieve the same functionality, so feel free to close this issue if you feel this is not relevant or less important than other. I'll do some more test and investigations on this as soon as I have time and I'll report my finding.

Thanks,
~Q
